.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -28,33 +22,15 @@ Please delete options that are not relevant.
 
 ## How Has This Been Tested?
 
-Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
-
-- [ ] Test A
-- [ ] Test B
-
-**Test Configuration**:
-* Firmware version:
-* Hardware:
-* Toolchain:
-* SDK:
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce it when relevant.
 
 ## PR checklist:
 
 - [ ] This comment contains a description of changes (with reason).
-- [ ] My code follows the style guidelines of this project
-- [ ] Make sure your code lints, is well-formatted and type checks
 - [ ] I have performed a self-review of my own code
-- [ ] I have commented my code, particularly in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
 - [ ] My changes generate no new warnings
 - [ ] I have added tests that prove my fix is effective or that my feature works
-- [ ] If you've added a new stage - have you followed the pipeline CLI conventions in the [USAGE guide](../USAGE.md)
-- [ ] [README.md](./README.md) is updated
-- [ ] If a new tool or package is included, update poetry.lock, [the conda recipe](../conda-recipe/pixelator/meta.yaml) and [cited it properly](../CITATIONS.md)
-- [ ] Include any new [authors/contributors](../AUTHORS.md)
-- [ ] New and existing unit tests pass locally with my changes
```

Comment:
I think it's implicit that the local test have passed. :) Also since all tests are run in CI there is really no risk that we merge code where the tests are passing.